acro/stata_config.py

```diff
@@ -0,0 +1,8 @@
+"""
+Config file to hold global variable for acro object
+accessible from acro files and stata
+mutable hence use of lower case naming
+Jim Smith 2023.
+"""
+
+stata_acro = "empty"  # pylint:disable=invalid-name
```

Comment:
Assuming that the str "empty" is never actually used, then changing this to the following will fix the remaining mypy issues:

```python
import acro
stata_acro: acro.ACRO
```

Reply:
done